frontend/src/components/LandingPage/LandingPageBody.module.css

```diff
@@ -0,0 +1,6 @@
+.tempText {
```

Comment:
nit: generally css standardizes on kebab case 

if you want to go deeper, see: https://getbem.com/naming/

Reply:
Ok will change to kebab case